Dockerfile-linux.template

```
@@ -318,8 +318,10 @@ RUN set -eux; \
 # https://github.com/docker-library/php/issues/822
 		--with-pic \
 		\
-# --enable-ftp is included here because ftp_ssl_connect() needs ftp to be compiled statically (see https://github.com/docker-library/php/issues/236)
+{{ if [ "8.1", "8.2", "8.3" ] | index(env.version | rtrimstr("-rc")) then ( -}}
```

How often is the `ftp` module used? I know this is extra conservative and so will only drop the module in `8.4` and above.  Would it make sense to change at the next patch releases, like `8.1.28`, `8.2.15`, and `8.3.2` or would there be too much breakage? Or maybe just some of them, like `8.3.2` and above?

---

Yes, this is extra conservative. I would expect the FTP extension to be used primarily in legacy code. Personally I've not used it ever.

GitHub's code search for `ftp_connect` reports 8.6k results (with some of them being the php-src stubs and some of them not being the actual `ftp_connect` function of the FTP extension).

---

Agreed, let's still be conservative for the current versions, but slightly more aggressive on even the next patch releases (the mitigation for the subset of affected users is pretty trivial); maybe something like this:

```suggestion
{{ if .version as $v | [ "8.1.27", "8.2.14", "8.3.1" ] | index($v) then ( -}}
```

:heart:

---

Applied.